docs/sources/panels/inspect-panel.md

```diff
@@ -0,0 +1,87 @@
++++
+title = "Inspect a panel"
+type = "docs"
+[menu.docs]
+identifier = "inspect-a-panel"
+parent = "panels"
+weight = 400
++++
+
+# Inspect a panel
+
+> **Note:** This documentation refers to a feature only available in Grafana 7.0 beta.
+
+The panel inspector helps you understand and troubleshoot your panels. You can inspect the raw data for any Grafana panel, export that data to a comma-separated variable (CSV) file, view query requests, and export panel and data JSON.
```

Comment:
why are we saying export data twice? 

```suggestion
The panel inspector helps you understand and troubleshoot your panels. You can inspect the raw data for any Grafana panel, export that data to a comma-separated values (CSV) file, view query requests & stats, view and edit panel JSON definition. 
```

Reply:
Because you can export different things in different ways. You can export raw data to a CSV file. You can also copy and paste JSON models to use in other apps.